invenio_cli/helpers/commands.py

```diff
@@ -228,6 +228,15 @@ def demo(self):
         command = ['pipenv', 'run', 'invenio', 'rdm-records', 'demo']
         subprocess.run(command, check=True)
 
+    def stop(self):
+        """Stops containers."""
+        docker_helper = DockerHelper(
```

Comment:
LGTM. This is more a comment for future work:

We should try to find a common way of using `docker_helper`. The containerize command has an instance as attribute and the local commands instantiate it each time is needed. It *might* be able to be refactored into something common.